manifests/apiserver.yaml

```diff
@@ -169,6 +169,8 @@ rules:
       - caliconodestatuses
       - ipamconfigs
       - tiers
+      - stagednetworkpolicies
+      - stagedglobalnetworkpolicies
```

Comment:
If I understand correctly, there also should be stagedkubernetesnetworkpolicies

Reply:
Yep, good catch!